### ISSUE TITLE
Poor video quality (blurriness) is observed when downloading YouTube videos via playlist download functionality.

### ISSUE THREAD

**Issue by ccvv3655** [aandrew-me/ytDownloader#266]
This software is great for downloading YouTube playlists; I've downloaded high-definition and 4K videos successfully. When I select the best quality option, it downloads clear WebM videos. However, if I manually select 4K and MP4, the downloaded videos are very blurry. I hope you can optimize the clarity of MP4 videos downloaded from playlists. Thanks!

**Comment by aandrew-me** [comment]
Yeah it looks like it downloads videos of lower resolution instead. I shall fix it

**Comment by aandrew-me** [comment]
Check in latest update
